src/app/shared/components/footer/footer.component.html

```diff
@@ -2,19 +2,79 @@
   <a class="flex items-center">
     <app-icon-logo [size]="23" />
   </a>
-  <span class="text-sm text-white font-bold flex items-center space-x-1">
+  <span class="text-sm text-gray-400 font-bold flex items-center space-x-1">
     <span>made with</span>
     <svg
       xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 20 20"
-      fill="white"
-      class="w-5 h-5"
+      viewBox="0 0 24 24"
+      fill="currentColor"
```

Comment:
```suggestion
      fill="#7AAB8A"
```

finds sehr cool mit dem hero icon; in HF grün aber auch:

<img width="942" alt="Screenshot 2024-09-30 at 21 38 54" src="https://github.com/user-attachments/assets/ef518b44-6dd6-4d6e-9fb3-607374f23989">

Reply:
Das farbige verwirrt etwas, da es zu viel Aufmerksamkeit auf sich zieht

![image](https://github.com/user-attachments/assets/74a4c60a-4d1b-48d7-8e6e-3f84611f1af4)
![image](https://github.com/user-attachments/assets/3f6a0688-5f03-4369-a8ea-f1dd283ef7b1)

Reply:
Impressum und Datenschutz werden noch später verlinkt